tools/toolchains/__init__.py

```diff
@@ -849,6 +849,21 @@ def compile_command(self, source, object, includes):
 
         return None
 
+    def check_toolchain_path(self, name):
+        """Check if the path to toolchain is valid. Exit if not.
+
+        Positional arguments:
+        name -- the name of the toolchain, as used when creating compile
+                commands to execute. Valid key for TOOLCHAIN_PATHS.
+
+        No return value but causes a system exit of execution if the path
+        does not exist.
+        """
```

Comment:
@bridadan do you think it is breaking change. I see that the build has failed while trying to get GCC_ARM toolchain path. I see
```
17:13:53 + mbed-cli test --compile --test-spec test_spec.json -vv -t GCC_ARM -m K64F
17:14:19 [ERROR] Toolchain path does not exist for GCC_ARM.
```
in http://e108747.cambridge.arm.com:8080/job/mbed-os/job/mbed-os-pr-build-parameterized/612/console